students/GABLE_SIMONE.md

```diff
@@ -0,0 +1,5 @@
+Title: Peto’s Paradox: how has evolution solved the problem of cancer evolution?
```

Comment:
Well done Simone.
You can use headings by starting lines with # (level 1):
# Heading 1
Or ## (level 2):
## Heading 2
Or ### (level 3):
### Heading 3